docs/tutorials/hello-world-application/creating-the-database.mdx

```diff
@@ -30,7 +32,9 @@ Add a new network by clicking on the `Default Workspace` dropdown in the Kuneifo
 </div>
 <br />
 
-Next, add the network details in the popup Menu. After you click "OK", the IDE will automatically switch to the new network.
+Next, add the network details in the popup Menu. You can find the `chain_id` in under `~/usr/.kwild/abci/config/genesis.json`, or you can leave the Chain ID empty and the IDE will retrieve it from the local Kwil node.
```

Comment:
I figure it would be `~usr/.kwild/...` or something else like `{root_dir}/...`.  Probably `~/usr/...` is wrong unless that's a Mac thing.

Reply:
I think i have seen it both ways, but it is probably cleanest to just do `{root_dir}/...`. I'll make that change.